rules/linux/auditd/lnx_find_cred_in_files.yml

```diff
@@ -0,0 +1,24 @@
+title: Credentials In Files

```

Comment:
A weird error here with the newline -.- 
Could you please rebase your PR and add only your commit to it? Here is an [instruction](https://stackoverflow.com/questions/34027850/how-to-pull-request-a-specific-commit). 
Thank you!